crest/Assets/Crest/Crest/Shaders/Ocean.shader

```diff
@@ -540,11 +528,27 @@ Shader "Crest/Ocean"
 				// Foam - underwater bubbles and whitefoam
 				half3 bubbleCol = (half3)0.;
 				#if _FOAM_ON
+				float foam = 0.0;
+				// Data that needs to be sampled at the undisplaced position
+				if (wt_smallerLod > 0.001)
+				{
+					const float3 uv_slice_smallerLod = WorldToUV(input.lodAlpha_worldXZUndisplaced_oceanDepth.yz, cascadeData0, _LD_SliceIndex);
```

Comment:
@daleeidd  Late feedback but it may be worth moving this up into the normal sampling code above. That would mean one set of of statements instead of two, and the UV would be computed once instead of twice.